packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -77,20 +77,15 @@ class DebuggerScreen extends Screen {
           ..disabled = true;
 
     final PButton pauseButton =
-        PButton.icon('Pause', FlutterIcons.pause_black_2x)..small();
+        PButton.icon('Pause', FlutterIcons.pause_black_2x, invertDark: true)
```

Comment:
make invertDark a property of the Icon not of the PButton.icon call.

Reply:
Done!